src/Test/Lint.hs

```diff
@@ -94,11 +94,11 @@ main = shaken test $ \args obj -> do
         need [obj "tracker-source.c", obj "tracker-source.h"]
         cmd "gcc" ["-c", obj "tracker-source.c", "-o", out]
     where gen t f = unit $ if isWindows
-                           then cmd "cmd /c" ["echo" ++ t ++ " > " ++ f]
-                           else cmd Shell "echo" ["x", ">", f]
+                           then cmd "cmd /c" ["echo " ++ t ++ " >" ++ f]
+                           else cmd "sh -c" ["echo " ++ t ++ " >" ++ f]
```

Comment:
This one I wanted to discuss with you. Shell isn't right now working properly because the code injection happens in the fsatrace function. That is, Shell will execute the "fsatrace <output> -- whatever" command, but we need the "sh" part after "fsatrace". 

In any case, shouldn't the shell to use be left to the user? I mean, a user could want to have some parts of the build system use "cmd" and some others use "bash" or "zsh" or whatever. Maybe Shell could just disappear.